src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.html

```diff
@@ -353,7 +353,8 @@
               </a>
               <div class="form-group">
                 <div class="align-center">
-                  <button class="btn btn-primary" tabindex="6" id="btn-save-{{ i }}" style="display:none;" type="submit">
+                  <button class="btn btn-primary" tabindex="6" id="btn-save-{{ i }}" style="display:none;"
+                          (click)="onSubmitEditInstructor(instr, i)">
```

Comment:
@rrtheonlyone This regression is caused by me in #9463. The proper solution is to import `FormsModule` in this page's module.

Reply:
Okay updated accordingly!